exercises/sam_jenkins/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
+#include <fstream>
+#include <unordered_map>
+#include <iomanip>
+#include <vector>
+#include "mapFunctions.h"
+
+
+using std::ifstream; using std::string; using std::cout; using std::endl; using std::istreambuf_iterator; 
+using std::unordered_map; using std::vector; using std::ofstream;
```

Comment:
`using namespace std;` is simpler and clearer. As I mentioned to Andre, typically we just prefix standard library functions with `std::` to make it clear that is where they're from, but for this small project this is ok.

Also please use one line per using statement.